test/tools/integration/Makefile

```diff
@@ -1,4 +1,4 @@
-CIRCLE_BUILD_NUM ?= local
+CIRCLE_BUILD_NUM ?= local # TODO: what should this be for prow?
```

Comment:
This must be unique per build, otherwise the e2e test runs into concurrency issue. Does prow provide something like a build id that we can leverage here? Worst case we could use nanosecond timestamp or something the like